### ISSUE TITLE
Make game responsive

### ISSUE THREAD

**Issue by paulgoetze** [kennethlarsen/thecatgame#26]
At the moment the game is not playable on devices without a keyboard and only works on large resolution displays. 

Making it responsive and playable on all devices includes:

* [x] rescaling on viewport/size change
* adding inputs for touch devices: 
  - [x] jumping the cat on swiping up
  - [x] adding buttons/areas for loading time-travel batteries and traveling
* [x] scaling down sprites for small resolution displays

**Comment by kennethlarsen** [comment]
I think we have to pick a sweet spot for desktop. What is the minimum size we should support?

**Comment by paulgoetze** [comment]
Yes, I think that’s a good point. Currently we use 1280x800px on itch.io. I guess this should pretty much cover desktop and tablets. I’m also working on some full screen mode right now, so that it just scales to your viewport size even if it is smaller.

For smartphones we will have to scale down the sprites though, this is what I get in the firefox dev tools for an iPhone 6s:
![grafik](https://user-images.githubusercontent.com/1781347/34496229-0daa8fcc-eff9-11e7-95d6-909b728ac42a.png)

Not playable yet :/

**Comment by paulgoetze** [comment]
Yay. With the latest commits in #27 The full scene is now smoothly resizeable, to whatever resolution or screen size you have 😃. 

I took an HD reference resolution (1080x1920) which I used for setting the default proportions and scaled based on the height. Changing the screen width will just infinitely scale the landscape and alter the position of the cat.

![grafik](https://user-images.githubusercontent.com/1781347/36063622-7d104862-0e7f-11e8-86d0-0e8e196c10f5.png)
